index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Lab 06</title>
+  </head>
+  <body>
+
+  </body>
+    <script src="app.js"></script>
```

Comment:
@kaylynyuh this should be moved over and indented in the same spot as your closing body tag